### PR TITLE
Added skip test marker in "test_nodes_maintenance.py" and "test_disk_failures.py" for external mode clusters

### PR DESCRIPTION
**Pull request by pintojoy**
These tests need to be skipped for external mode cluster as per https://github.com/red-hat-storage/ocs-ci/issues/5720

### CODE REVIEW

**Comment by am-agrawa** [comment]
@pintojoy PR checks are failing, and please run the PR validation job targeting the relevant test cases once the code is complete.

**Review by am-agrawa — CHANGES_REQUESTED** [review]
`tests/manage/z_cluster/nodes/test_nodes_restart.py::TestNodesRestart::test_nodes_restart[True]`

Change for this test also needs to be incorporated. Check the linked issue.

**Review by prsurve — APPROVED** [review]

**Review by am-agrawa — DISMISSED** [review]
LGTM

**Comment by pintojoy** [comment]
PR validation job is failing at creation pod for sanity run post health check with error:
"Error is Error from server (Forbidden): error when creating "/tmp/POD_xls5smf4": pods "pod-test-rbd-094623ad03ef4c0f8faa1646b34" is forbidden: error fetching namespace "namespace-test-81ec28e2829343c8800e44e20": unable to find annotation openshift.io/sa.scc.uid-range". 
Debugging this failure

File: ocs_ci/helpers/sanity_helpers.py [hunk 1]
```diff
@@ -183,3 +187,4 @@ def __init__(self):
         self.pod_objs = list()
         self.obc_objs = list()
         self.ceph_cluster = CephClusterExternal()
+        self.ceph_cluster_external = CephClusterExternal()
```
Comment (sidhant-agrawal) [inline ocs_ci/helpers/sanity_helpers.py:190]: Why do we need to create this duplicate? We already have `ceph_cluster` from above line.

**Comment by pintojoy** [comment]
tests/manage/z_cluster/nodes/test_nodes_restart.py::TestNodesRestart::test_nodes_restart[True] test is verified locally and the verification log can be found at http://pastebin.test.redhat.com/1054611

File: ocs_ci/helpers/sanity_helpers.py (hunk 1 above)
Comment (am-agrawa) [reply ocs_ci/helpers/sanity_helpers.py]: @pintojoy Would you be able to answer this?

**Review by am-agrawa — DISMISSED** [review]

File: ocs_ci/helpers/sanity_helpers.py (hunk 1 above)
Comment (pintojoy) [reply ocs_ci/helpers/sanity_helpers.py]: @am-agrawa I have taken care of it in latest changes

File: tests/manage/z_cluster/nodes/test_nodes_restart.py [hunk 2]
```diff
@@ -71,6 +72,7 @@ def test_nodes_restart(
         """
         ocp_nodes = get_node_objs()
         nodes.restart_nodes_by_stop_and_start(nodes=ocp_nodes, force=force)
+        time.sleep(300)
         self.sanity_helpers.health_check()
```
Comment (keesturam) [inline tests/manage/z_cluster/nodes/test_nodes_restart.py:76]: Please change `restart_nodes_by_stop_and_start` to pass the timeout value into the function itself.

**Comment by pintojoy** [comment]
This PR is addressed by other subsequent PRs (PR #6344 and PR #6256), Hence closing the PR.

**Pull request closed without merging**